src/app/utils/request.ts

```diff
@@ -1,5 +1,14 @@
 import { Request } from 'express'
 
-export const getRequestIp = (req: Request) => {
+interface IRequestWithId extends Request {
```

Comment:
nit: Can we extend the namespace globally (following suggestions in this [SO post](https://stackoverflow.com/questions/37377731/extend-express-request-object-using-typescript), some may be outdated) so it's consistent throughout the application?

Can also be done in a different PR, since I think we need to extend request to also have `req.session.user`

Reply:
Ok! Will create issue for this and work on ti